dockerfile.dev

```diff
@@ -0,0 +1,26 @@
+# Use the official python 2 base image
+FROM python:2
```

Comment:
How about using an alpine based docker image for python here?

Reply:
@Ugtan I went through this article and the reasoning behind not using `alpine` [link](https://pythonspeed.com/articles/alpine-docker-python/). thoughts?

Reply:
@kmehant I guess it is better not to use alpine for the reason mentioned in the article. Thanks for the article tho.